pkgs/misc/kitty-themes/default.nix

```
@@ -0,0 +1,28 @@
+{
+  fetchFromGitHub, lib, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "kitty-themes";
+  version = "2021-01-01";
```

```suggestion
  version = "2022-01-01";
```
The commit below is from January 2022 and not January 2021. Please also update the title of the pull request and the commit message.

---

I just saw, that the commit message is already correct.

---

@Luflosi my apologies; updated